### ISSUE TITLE
Stop extending event emmiter

### ISSUE THREAD

**Issue by hipstersmoothie** [oliver-moran/jimp#531]
It seems weird to me that we extend a event emmiter. IDK how many people use the emitted event but it seems more like hinderance than a feature to me.

**Comment by edi9999** [comment]
Yes, extending event emitter is probably not a used feature, just legacy.
